### ISSUE TITLE
Suggest UI Designs and Features

### ISSUE THREAD

**Issue by infiniteoverflow** [infiniteoverflow/Flutter-Tutorial-App#6]
Feel free to suggest your ideas on the UI and features !!

**Comment by gargk747** [comment]
Hey @infiniteoverflow I want to try some UI changes and will try to add some features in this application.
Can you please  assign me this issue?

**Comment by avishmehta68710** [comment]
i woud like to add some background images and colour and make the buttons look good. can you please assign it to me

**Comment by avishmehta68710** [comment]
would also like to add a form feature for submission purposes. please assign the task to me

**Comment by infiniteoverflow** [comment]
@gargk747 Kindly add a ui design that you have in mind. Based on that i will create an issue and assign it to you with the labels.

**Comment by infiniteoverflow** [comment]
@avishmehta68710 Can u please explain the purpose of the form feature. I couldnt understand that.

**Comment by Saransh-cpp** [comment]
@infiniteoverflow right now the intro screen and the splash screen are same, we can do something like a 3 page intro slider which will be visible only once with some information about the app and then a splash screen which will be visible every time the user starts the app and it will contain only the name or the logo. The splash screen won't have any buttons, user will be redirected to home page automatically. Should I open 2 separate issues for this?

**Comment by infiniteoverflow** [comment]
Yes , so that would be good. So we already have an issue to show firstTime screen , u can work on that after the following part. And if FirstTime , then user should be redirected to the IntroductionScreen , otherwise the HomePage.

You may need assets for the Introduction screen , i will find some good ones and add it to the assets folder. Meanwhile why dont you work on showing the Splash screen on load-up...just show the Flutter logo for now , later i will find a good asset and update it. I will create an issue for it and assign it to u.

After the PR for splash screen you can work on the Introduction screen.

**Comment by infiniteoverflow** [comment]
@Saransh-cpp Kindly comment on the issue so that i can assign it to u.

**Comment by Saransh-cpp** [comment]
@infiniteoverflow till the time you add relevant pictures for the intro screen can I start working on the UI for login and sign-up screen? and also are you thinking of integrating the app with firebase for sign in and login?

**Comment by infiniteoverflow** [comment]
@Saransh-cpp Actually i have already assigned someone else with that issue. If that person is not able to work on that , will assign it to u. And yes i am planning to integrate it with Firebase.

Regarding the assets , i am just caught up with some work , will upload it as soon as i can.

**Comment by Saransh-cpp** [comment]
> @Saransh-cpp Actually i have already assigned someone else with that issue. If that person is not able to work on that , will assign it to u. And yes i am planning to integrate it with Firebase.
> 
> Regarding the assets , i am just caught up with some work , will upload it as soon as i can.

Yes I just saw the other issue, no problem, I will work on some other features then!
